### DISCUSSION TITLE
Habit-associated URIs or Notes

### DISCUSSION THREAD

**Discussion by finnmglas** [iSoron/uhabits#679] [Feature Requests] [answered]
Amazing app.

An idea for a feature:

Implement **notes** associated with habits. Useful user-specified activity-metadata could be stored there. Technically we can use the 'Question' field but that's kinda weird.

Similar, possibly even better Idea: Allow users to store **URIs** (select an app / a website) associated with habits. Use case: Users can select journaling, reading, news-related websites or Apps.

**Comment by tamarw** [comment]
Agreed, I need this. The only reason I'm still on 7 Weeks.

**Comment by hiqua** [comment, marked as answer]
Notes will come with the v2, see: https://github.com/iSoron/uhabits/releases/tag/v2.0.0-alpha

URIs might be a bit too specific to be really useful. At least in my case, I have only one habit I can complete using my phone. But anyway, these URIs could always be stored in the notes, right? At least as a first implementation.

**Reply by tamarw** [reply]
Sounds great, thanks @hiqua - I'm enrolled in the beta program on Google Play - any chance there's a v2 beta to try out on the horizon? I'm only getting public apps at the present.

For me, notes are great. I don't need URIs.
